lib/storage.js

```diff
@@ -23,6 +18,11 @@ bedrock.events.on('bedrock-mongodb.ready', async () => {
       background: false,
       expireAfterSeconds: 0
     }
+  }, {
+    // index for query that is used to look up tokens
+    collection: 'oauth2-client-accessToken',
+    fields: {'token.client_id': 1, 'token.token_endpoint': 1},
+    options: {unique: true, background: false}
```

Comment:
I'd move this index up first so it's the first one people read -- it's the most important one for understanding the data modeling. So this one first, then the expires one. We generally try to order indexes that way (or if there are many, perhaps alphabetically -- but either way, this second one should be first).